orca/templates/db/resources/usr/local/bin/mysql_init.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+set -e
+
+# Inspired by Percona's MySQL docker image, might share code snippets
+
+# shellcheck disable=SC1091
+. _liferay_common.sh
+
+function check_setup {
+	block_begin "Check if MySQL set up"
+
+	if [ -d /var/lib/mysql/data/mysql ];
+	then
+		block_finish "Check if MySQL is set up: DONE"
+
+		exit 0
+	else
+		block_finish "Check if MySQL is set up: NOT DONE"
+	fi
+}
+
+function datadir_init {
+	block_begin "Datadir initialization"
+
+	install -v -o mysql -g mysql -m 0700 -d "${HOME}/data" "${HOME}/log"
+
+	mysqld --initialize-insecure
+
+	block_finish "Datadir initialization"
+}
+
+function first_run_start {
+	block_begin "Initialize the 'mysql' database, first run"
+
+	mysqld --skip-networking &
+	pid="$!"
+
+	for second in {120..0}; do
+		msg "MySQL init process in progress... ${second} seconds left"
+
+		if echo 'SELECT 1' | mysql &> /dev/null;
+		then
+			break
+		fi
+
+		sleep 1
+	done
+
+	if [ "${second}" = 0 ];
+	then
+		fail "MySQL init process failed."
+		exit 1
+	fi
+	block_finish "Initialize the 'mysql' database, first run"
+}
+
+function first_run_stop {
+	block_begin "Shut down the initializator mysqld instance"
+
+	msg "kill -s TERM $pid"
+
+	if ! kill -s TERM "$pid" || ! wait "$pid";
```

Comment:
Please add ()s and ${pid}

Reply:
Done.

Reply:
Does not work properly if `()` is added, I could not figure out the exact reason so far.
I left it as is for now.